lib/config.js

```diff
@@ -43,8 +43,10 @@ config.serverPort = '3000';
 config.sslCertificateFile = '/etc/pki/tls/certs/localhost.crt';
 config.sslKeyFile = '/etc/pki/tls/private/localhost.key';
 config.sslCAFile = '/etc/pki/tls/certs/server-chain.crt';
-config.fileUploadMaxBytes = 1e7;
+config.fileUploadMaxBytes = 1e8;
 config.fileUploadMaxParts = 1000;
+config.fileUploadS3Bucket = 'prairielearn-dev';
+config.fileUploadS3MinBytes = 1e7;
```

Comment:
Small files can currently go in the database, as PL already demonstrates. 

We can decide to support small files (ie. < 200kb) or we can upload all files in S3 if we would like. Any thoughts on which approach is preferable?